string object_name
---
float64[] object_state
